loops-arrays-objects.js

```diff
@@ -114,6 +127,24 @@ const students = [
     {name: 'Karima', city: 'Utrecht', zipCode: '3531', neighborhood: null},
 ];
 
+const neighborhood = {
+    '3513': 'Pijlsweerd',
+    '3514': 'Vogelenbuurt',
+    '3512': 'Binnenstad',
+    '3531': 'Lombok',
+    '3572': 'Wittevrouwen',
+    '3581': 'Oudwijk',
+    '3583': 'Schildersbuurt'
+};
+
+students.forEach(student => {
+    student.neighborhood = neighborhood[student.zipCode];
+})
+
+console.log(students);
+
```

Comment:
Mooie uitwerking, maar hier heb je een functie gebruikt in plaats van een loop, if or switch. De code is zo wel lekker efficient. Goed gedaan!